### PR TITLE
Override privacy for collaborators list

### PR DESCRIPTION
**Pull request by sbose78**
If the collaborators API is called using a service account token, then override the email privacy settings.

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-291-2`

File: controller/collaborators.go [hunk 1]
```diff
@@ -43,6 +44,8 @@ func NewCollaboratorsController(service *goa.Service, db application.DB, config
 
 // List collaborators for the given space ID.
 func (c *CollaboratorsController) List(ctx *app.ListCollaboratorsContext) error {
+	isServiceAccount := token.IsSpecificServiceAccount(ctx, "fabric8-notification")
```
Comment (alexeykazakov) [inline controller/collaborators.go]: Please use tenant.Notification instead of hardcoded "fabric8-notification".

**Comment by sbose78** [comment]
Sure! Was trying things out - and then had to shut my laptop and run for dinner with PMs :P

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-291-3`

**Comment by sbose78** [comment]
@alexeykazakov , test added.
This is ready for review.

**Comment by fabric8cd** [comment]
@sbose78 snapshot fabric8-auth image is available for testing.  `docker pull docker.io/fabric8/fabric8-auth:SNAPSHOT-PR-291-4`

**Pull request merged**